docs/en/configurations/pre-processors.md

```diff
@@ -37,6 +37,29 @@ Contrary to what its name indicates, [*sass*-loader](https://github.com/jtangeld
 
 See the [Advanced Loader Configuration](./advanced.md) Section for further information about how to configure vue-loader.
 
+#### Loading a global settings file
+
+A common request is to be able to load a settings file in each component without the need to explicity import it each time, e.g. to use scss variables globally throughout all components. To accomplish this:
+
+``` bash
+npm install sass-resources-loader --save-dev
+```
+
+Then update your `build/utils.js` file:
```

Comment:
Let me know if that reads better, I guess I incorrectly assumed most people would be use the webpack template!